webcompat/webhooks/model.py

```diff
@@ -234,19 +242,20 @@ def get_public_issue_number(self):
         return url
 
     def classify(self):
-        """Make a request to bugbug and label the issue.
+        """Make a request to bugbug and milestone the issue.
 
-        Gets issue classification from bugbug and labels
+        Gets issue classification from bugbug and milestones
         the issue if probability is high
         """
         data = get_issue_classification(self.number)
         needsdiagnosis_false = data.get('class')
         proba = data.get('prob')
 
         if needsdiagnosis_false and proba and proba[1] > THRESHOLD:
-            payload = {'labels': ['bugbug-probability-high']}
-            path = f'repos/{PRIVATE_REPO}/{self.number}/labels'
-            make_request('post', path, payload)
+            path = f'repos/{PRIVATE_REPO}/{self.number}'
+            self.milestone = AUTOCLOSED_MILESTONE_ID
+            payload_request = {'milestone': self.milestone}
```

Comment:
Maybe these two lines can be one. 
```suggestion
            payload_request = {'milestone': AUTOCLOSED_MILESTONE_ID}
```